cmd/zpool/zpool_main.c

```diff
@@ -1766,7 +1766,7 @@ print_import_config(const char *name, nvlist_t *nv, int namewidth, int depth,
  */
 static void
 print_logs(status_cbdata_t *cb, zpool_handle_t *zhp, nvlist_t *nv,
-    int namewidth, boolean_t verbose, int name_flags)
+    int namewidth, int name_flags)
```

Comment:
I 2nd this.  I'd love to see all the one-off args inside cbdata:

`print_logs(zpool_handle_t *zhp, nvlist_t *nv, status_cbdata_t *cb)`
...
```
cb->verbose
cb->name_flags
cb->namewidth
```